schema.json

```diff
@@ -696,6 +696,17 @@
     ]
   },
   {
+    "name": "Categories",
+    "settings": [
+      {
+        "type": "checkbox",

```

Comment:
Can you change this to a dropdown or radios? The idea is to have multiple designs for the navigation, if in the future we add a new style we can add a line to the dropdown.

So, instead of a boolean, we should use a string setting with 2 possible values